tool/tsh/db.go

```diff
@@ -1119,6 +1119,20 @@ func withConnectRequirements(tc *client.TeleportClient, route *tlsca.RouteToData
 			// (elasticsearch-sql-cli cannot be configured to use specific certs).
 			r.requireLocalProxyWithTunnel(formatDBProtocolReason(route.Protocol))
 		}
+		if r.localProxy && r.tunnel {
+			return
+		}
```

Comment:
At first, without having a full context of `withConnectRequirements`, it was hard for me to understand why we're returning early here. I guess if localProxy and tunnel are already requested then there's no more things that we could possibly request? Could you add a comment explaining that?

Reply:
When local proxy tunnel is already required, we return early here to avoid an extra API call for fetching the MFA requirement. I used an early return to reduce the indentation of the code below. I'll add a comment explaining this